image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -316,6 +316,7 @@ defaults:
       image:
         name: [< required "" .ScannerV4DBImageRemote >]
         tag: [< required "" .ScannerV4ImageTag >]
+  exposeMonitoring: false
```

Comment:
So, as a good YAML reviewer I have counted whitespace characters and came to the conclusion that we seem to be lacking 50% of initial whitespaces here?

Reply:
great catch